v20.2/resume-schedules.md

```diff
@@ -0,0 +1,82 @@
+---
+title: RESUME SCHEDULES
```

Comment:
A future enhancement -- talk about how `RESUME SCHEDULE` handles the `next_run` time. AFAIK it just sets it to be the next timestamp that conforms to the schedule but we'd want to confirm.